x-pack/plugins/apm/public/context/breadcrumbs/use_breadcrumb.ts

```diff
@@ -21,17 +21,17 @@ export function useBreadcrumb(breadcrumb: Breadcrumb | Breadcrumb[]) {
 
   const matchedRoute = useRef(match?.route);
 
-  if (matchedRoute.current && matchedRoute.current !== match?.route) {
-    api.unset(matchedRoute.current);
-  }
+  useEffect(() => {
```

Comment:
Is this related to storybook bump?

Reply:
hey @gbamparop, yes, it is, https://buildkite.com/elastic/kibana-pull-request/builds/41690#5317592f-de5c-4750-9b11-79eace1dde2b